
---

string path_to_log 
